model/implementation/DeliveryExecutionStateService.php

```diff
@@ -57,6 +57,9 @@ class DeliveryExecutionStateService extends AbstractStateService implements \oat
     const OPTION_CANCELLATION_DELAY = 'cancellation_delay';
     const OPTION_TIME_HANDLING = 'time_handling';
 
+    const TIME_HANDLING_EXTRA_TIME = 'extra_time';
+    const TIME_HANDLING_TIMER_ADJUSTMENT = 'timer_adjustment';
```

Comment:
Why do we need these constants if they are not used?

Reply:
I've updated the updater to use the constant `TIME_HANDLING_EXTRA_TIME`, and client extensions which require the timer adjustment feature will use the second constant `TIME_HANDLING_TIMER_ADJUSTMENT`.